### PR TITLE
[#25] Proper integration of OCR with service API, with meaningful image data. 

### PR DESCRIPTION
**Pull request by lancelafontaine**
[#25] 

- [x] Integrates proper image data decoding into API with base64-encoded png data that would be sent from front-end

### CODE REVIEW

File: src/task_service/README.md [hunk 1]
```diff
@@ -10,5 +10,39 @@ export COMPOSE_FILE=ci
...
...nDWOqXaNwAIljvb963W3A6gUu/TEwARjm2vqLRBTwYA0PXjwX/1Mpq2CmOKLHM7n20/o+RCKkoAPPUaPcfshpjjYw0NAIRUhABY9motfTtVKoT5knTK4s9jWNUj9uzJK73BE0AgkxHOK8AFsc2A1b2G/br0xQqZAMCXjq23I2MwkAD8udI2CuvyoCcqBzey2kolekxt0ErRhtWzgK6p8uCu9sV2onJidqeUfEsSJSvxS27DIHRtr/3RAEJri/lsOioGrYYYqU7mi4VpLuazyajfrmcjfgDwR6ujuWOh1WA5wep0L72x1TuqP9mdmaZpziftNOtnWRIAyGh1NBqOZ6Zpmot2FMDti7tR3g8AZLQ5mc/H1eiBjlMvKfm0HiIAgAgVh7OFaZrmYtovhvzb/t2tjoKBCGbb4+l8Phs34wEy2pyduLjreq3SdHRFa32dRqaHtjXPkgCBQIDYfOBnztpxPwCZ7K+XPF6AA5vG22Ixn037eWvVo83pfL5Y7HRibH1EGEh2raLMukk/gH/9T9NcbzmAoPcW22wuF466OCG7k0u+StCqFyCzQ7fJCL0ujProjVgfOu8jCNIfYIPRdLk5nDqOorN2MkBuRjwhyAAbbzo+k54Pi0HSOsD6SYLwR4rDmTlrx/0EAJDBdHvaTQf8thT8bLRuixOTeoTcRIJQdXy1klsWk3Z2VT6C9JNABKLOAd/mo2rUv03PHynaTxwOLL5wrBeQ/lB1MqlG/I7aSnYXw7x9NsLPFkemaX3pzxbH5nyYD5F+NhKPR1gSyNDOqIEHCnB007hZdZDgZndEgGk3GyQJko1EIwECyFC2O7VPnqxHt/P6bn41qKCHvS4FjmR3TslNc6/rBYTe1E+maR7YHRB6ZwxNlmXNoDme377Mbug6UKfdeNZVSVIMmg/z9MvcjjU0RZZV3ecsoW26rsiSqlMMz3MuRTi2+EVFUkUZwquR5g1VlhSd5sNuub9QAU4rpSZLsmb4GF7Y+aADAEBXBqXEL0ppLmauU6jD2Z2bmCrJBhfmqCsUDKHnwaiPEPoBaI0w08tpgysFfYR+VPi9PkLovdPFQqxCNSQM+Qgdg1EfIfS+6Z1cQS+JPRzLDqHj8A4/QgghdCvwe32EEELoVmDURwghhG4FRn2EEELoVmDURwghhG7F/we6f7mxCZ+52QAAAABJRU5ErkJggg=="
```
Comment (Vynny) [inline src/task_service/README.md:27]: This isn't good practice at all. Base64 can significantly increase the size of even a small image. Why not simply take the code we have for uploading the lease on the backend and use that here?

**Review by Vynny — CHANGES_REQUESTED** [review]

File: src/task_service/README.md (hunk 1 above)
Comment (lancelafontaine) [reply src/task_service/README.md:27]: As far as I'm aware, representing image data in base64 is pretty standard on the frontend. Placing ☝️ in `` displays the image.

You're right about using whatever functionality backend has though.

Comment (Vynny) [reply src/task_service/README.md:27]: Representing thumbnails and other small images in base64 is standard, not large images.
In any case, just rip the file upload stuff from backend and use multipart upload for the ocr, makes it easier to set stuff like file upload limits and specify filetype support.

Comment (dodoels) [reply src/task_service/README.md:27]: I agree with Vynny, usually you only base64 the small images

**Pull request closed without merging**